install/netbox-install.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: bvdberg01
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get update
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y apache2
+$STD apt-get install -y redis-server
+$STD apt-get install -y postgresql
+$STD apt-get install -y python3
+$STD apt-get install -y python3-pip
+$STD apt-get install -y python3-venv
+$STD apt-get install -y python3-dev
+$STD apt-get install -y build-essential
+$STD apt-get install -y libxml2-dev
+$STD apt-get install -y libxslt1-dev
+$STD apt-get install -y libffi-dev
+$STD apt-get install -y libpq-dev
+$STD apt-get install -y libssl-dev
+$STD apt-get install -y zlib1g-dev
```

Comment:
This is smoother


$STD apt-get install -y \
  curl \
  sudo \
  mc \
  apache2 \
  redis-server \
  postgresql \
  python3 \
  python3-pip \
  python3-venv \
  python3-dev \
  build-essential \
  libxml2-dev \
  libxslt1-dev \
  libffi-dev \
  libpq-dev \
  libssl-dev \
  zlib1g-dev